packages/adblocker/test/engine/engine.test.ts

```diff
@@ -703,16 +720,35 @@ $csp=baz,domain=bar.com
     describe('script injections', () => {
       it('injects script', () => {
         const engine = Engine.parse('foo.com##+js(script.js,arg1)');
-        engine.resources = Resources.parse('script.js application/javascript\n{{1}}', {
-          checksum: '',
-        });
+        engine.resources = Resources.parse(
+          JSON.stringify({
+            redirects: [],
+            scriptlets: [
+              {
+                names: ['script.js'],
+                content: 'function script() {}',
+                dependencies: [],
+                executionWorld: 'main',
+                requiresTrust: false,
+              },
+            ],
+          } as ResourcesDistribution),
```

Comment:
Editor complains:

```
Conversion of type '{ scriptlets: { names: string[]; content: string; fnName: string; dependencies: never[]; executionWorld: "MAIN"; requiresTrust: false; }[]; }' to type 'ResourcesDistribution' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Property 'redirects' is missing in type '{ scriptlets: { names: string[]; content: string; fnName: string; dependencies: never[]; executionWorld: "MAIN"; requiresTrust: false; }[]; }' but required in type 'ResourcesDistribution'.ts(2352)
```